### ISSUE TITLE
Code completion does not work for a crate - "File is not included in module tree, analysis is not available" - Reproducible

### ISSUE THREAD

**Issue by YenForYang** [intellij-rust/intellij-rust#4679]
## Environment

* **IntelliJ Rust plugin version:**
0.2.110.2150-192
* **Rust toolchain version:**
1.39.0
* **IDE name and version:**
Clion 2019.2.5
* **Operating system:**
Windows 10 Version 1909

## Problem description
Should be easy to reproduce.

Code completion for `reqwest` does not work when using `reqwest = "0.10.0-alpha.2"` under `[dependencies]` in `Cargo.toml` (the latest version of reqwest).

Whats interesting is that it doesn't work for version "0.10.0-alpha.1" either, but it **does** work for "0.9.22" (the last 0.9 release). Unfortunately I can't use 0.9.22 as there are breaking changes in 0.10+

![Demo](https://user-images.githubusercontent.com/16462860/69485426-ea319f00-0e04-11ea-8a31-16d4c5645ff6.gif)

Note in the above GIF at the start (where the cursor is). `new()` in `ClientBuilder::new()` isn't blue.
## Steps to reproduce
Create a new Rust project in CLion, and

`Cargo.toml` add these dependencies:

```
...
[dependencies]
reqwest = "0.10.0-alpha.2"
tokio = "0.2.0-alpha.6"
```

in `main.rs` have something like:
```rust
use reqwest::ClientBuilder;

#[tokio::main]
pub async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let client = ClientBuilder::new()
        .http2_prior_knowledge()
...
```

**Comment by YenForYang** [comment]
Seems to be the same issue described in https://github.com/intellij-rust/intellij-rust/issues/4627#

**Comment by Undin** [comment]
@YenForYang thanks for the report!
The reason is module declaration is generated by macro call.
Try to enable experimental macro expansion engine (see #3628). It should solve your issue

> Seems to be the same issue described in #4627

The global reason is the same, yes. But in this case experimental engine can process it but in case of #4627 - not yet because `mod` declaration is under several macro calls that is not supported yet

**Comment by Undin** [comment]
It's fixed in #4914 and will be landed in the next release (I hope).
I close this issue as a duplicate of #4627 because they have the same reason
